### PR TITLE
Change tiny declared globals to local scope.

### PR DESCRIPTION
**Pull request by bucaran**
Change tiny declared globals to local scope. Better not to pollute the global space.

### CODE REVIEW

File: plugins/tiny/tiny.fish [hunk 1]
```diff
@@ -68,6 +62,12 @@ function github.io -a url code
 end
 
 function tiny -d "get a git.io short URL"
+  set -l __github_url         "https://github.com"
+  set -l __github_user        (git config github.user)
+
+  set -l __TINY_ERR_BAD_URL   1
```
Comment (bpinto) [inline plugins/tiny/tiny.fish]: These variables need to follow a pattern.

> Variables in fish can be exported. This means the variable will be inherited by any commands started by fish. It is convention that exported variables are in uppercase and unexported variables are in lowercase.

P.S.: It's always good to wait for a code review before merging.

Comment (bobthecow) [inline plugins/tiny/tiny.fish]: Also double underscore seems weird for local variables.

Comment (bucaran) [inline plugins/tiny/tiny.fish]: I am not sure I _completely_ agree with that code convention. At least for error codes, it feels more natural to make them uppercase.

<div>-------- 元のメール --------</div><div>送信元： Bruno <notifications@github.com> </div><div>日時:2015/02/11  AM1:54  (GMT+09:00) </div><div>宛先： bpinto/oh-my-fish <oh-my-fish@noreply.github.com> </div><div>Cc： Jorge Bucaran <jbucaran@me.com> </div><div>件名: Re: [oh-my-fish] Change tiny declared globals to local scope. (#369) </div><div>
</div>In plugins/tiny/tiny.fish:

> @@ -68,6 +62,12 @@ function github.io -a url code
>  end
>  
>  function tiny -d "get a git.io short URL"
> +  set -l __github_url         "https://github.com"
> +  set -l __github_user        (git config github.user)
> +
> +  set -l __TINY_ERR_BAD_URL   1
These variables need to follow a pattern.

Variables in fish can be exported. This means the variable will be inherited by any commands started by fish. It is convention that exported variables are in uppercase and unexported variables are in lowercase.

P.S.: It's always good to wait for a code review before merging.

Comment (bucaran) [inline plugins/tiny/tiny.fish]: @bobthecow I agree double underscore for _local_ variables is weird. My bad. :sweat_smile:

@bpinto I am not sure I _completely_ agree with that code convention. At least for error codes, it feels more natural to make them uppercase.

Comment (bpinto) [inline plugins/tiny/tiny.fish]: I don't see any problems with having error codes downcased, I honestly don't enforce this pattern on plugins and themes.

If you want to follow your own convention, it's your call. But, personally, I don't think fighting with a language style guide is the way to go.

Comment (bucaran) [inline plugins/tiny/tiny.fish]: Me neither, looking at it now I have an urge to make them lowercase. 

Having a style guide is a cool move. I don't know of any other fish style guides out there, so we could _pioneer_ there.

**Pull request merged**